example/ios/example/GoogleAdsProvider.swift

```diff
@@ -83,48 +82,39 @@ import SpotImCore
             return kGADAdSizeMediumRectangle
         }
     }
+  
+    @objc public class func setSpotImSDKWithProvider() {
+      SpotIm.setGoogleAdsProvider(googleAdsProvider: GoogleAdsProvider())
+    }
```

Comment:
@oded-regev This code was added to GoogleAdsProvider file only in RN. We should update the files in the pod repo.